CONTRIBUTING.md

```
@@ -0,0 +1,257 @@
+# Contributing to TODO: Project Name
+
+⭐ First off, thank you for considering contributing to this project! ⭐
+
+We welcome contributions from everyone. By participating in this project, you agree to abide by our Code of Conduct.
+
+## � IMPORTANT: Discord Communication is Mandatory
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)